chemprop/featurizers/molgraph/cache.py

```diff
@@ -20,24 +20,24 @@ class MolGraphCacheFacade(Sequence[MolGraph], Generic[T]):
 
     Parameters
     ----------
-    inputs : Iterable[T]
+    inputs : Iterable[S]
         The inputs to be featurized.
     V_fs : Iterable[np.ndarray]
         The node features for each input.
     E_fs : Iterable[np.ndarray]
         The edge features for each input.
-    featurizer : MolGraphFeaturizer[T]
+    featurizer : MolGraphFeaturizer[S]
         The featurizer with which to generate the
         :class:`~chemprop.featurizers.molgraph.molgraph.MolGraph`\s.
```

Comment:
```suggestion
        :class:`~chemprop.data.molgraph.MolGraph`\s.
```
Also on lines 48, 70